Refactor: improve `rz-bin -L` output

PremadeS opened the pull request:
Added `author` + `license` cols
Removed `bin/xtr` col

 <!-- Filling this template is mandatory -->

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository.
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style).
- [ ] I've documented every `RZ_API` function and struct this PR changes.
- [ ] I've added tests that prove my changes are effective (required for changes to `RZ_API`).
- [ ] I've updated the [Rizin book](https://github.com/rizinorg/book) with the relevant information (if needed).

**Detailed description**

<!-- Explain the **details** for making this change. Is a new feature implemented? What existing problem does the pull request solve? How does the pull request solve these issues? Please provide enough information so that others can review your pull request. -->

removed the `bin/xtr` columns and put `author`, `license` and `version` in separate columns. 
note the `xtr` entries don't have `author` and `version` attributes
...
**Test plan**
<!-- What steps should the reviewer take to test your pull request? Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots/videos. This is your time to re-check that everything works and that you covered all the edge cases -->
run `rz-bin -L` (truncated output)
```bash
any          LGPL3     Dummy format rz_bin plugin             nibble          
art          LGPL3     Android Runtime                        pancake         
avr          LGPL3     ATmel AVR MCUs                         pancake         
bf           LGPL3     brainfuck                              pancake         
bflt         LGPL3     bFLT uClinux executable                Oscar Salvador  
bios         LGPL      BIOS bin plugin                        pancake         
bootimg      LGPL3     Android Boot Image                     pancake         
cgc          LGPL3     CGC format rz_bin plugin               ret2libc        
coff         LGPL3     COFF format rz_bin plugin              Fedor Sakharov  
dex          LGPL3     dex bin plugin                         deroad          
dmp64        LGPL3     Windows Crash Dump x64 rz_bin plugin   abcSup          
```
run `rizin -qc 'Lit'` (truncated table output)
```bash
name         license description                          author         version 
---------------------------------------------------------------------------------
any          LGPL3   Dummy format rz_bin plugin           nibble         
art          LGPL3   Android Runtime                      pancake        
avr          LGPL3   ATmel AVR MCUs                       pancake        
bf           LGPL3   brainfuck                            pancake        
bflt         LGPL3   bFLT uClinux executable              Oscar Salvador 
bios         LGPL    BIOS bin plugin                      pancake        
bootimg      LGPL3   Android Boot Image                   pancake        
cgc          LGPL3   CGC format rz_bin plugin             ret2libc       
coff         LGPL3   COFF format rz_bin plugin            Fedor Sakharov 
dex          LGPL3   dex bin plugin                       deroad         
dmp64        LGPL3   Windows Crash Dump x64 rz_bin plugin abcSup         
```
...

**Closing issues**
<!-- put "closes #XXXX" in your comment to auto-close the issue that your PR fixes (if any). -->
closes #5181 
...

PremadeS committed:
Refactor: improve `rz-bin -L` output

Added `author` + `license` cols
Removed `bin/xtr` col

PremadeS commented:
need to update tests, though if any reviewer sees this in the mean time should I stick with the new output format or the one similar to `rz-asm -L` ?

notxvilka reviewed (changes requested):
Please add missing authors in corresponding plugins based on the SPDX headers and git history

notxvilka commented on librz/core/cbin.c:
```diff
@@ -4996,7 +4996,7 @@ RZ_IPI RzCmdStatus rz_core_binxtr_plugin_print(const RzBinXtrPlugin *bx, RzCmdSt
 	case RZ_OUTPUT_MODE_STANDARD:
 		name = strncmp(bx->name, "xtr.", 4) ? bx->name : bx->name + 3;
-		rz_cons_printf("xtr  %-11s %s (%s)\n", name,
+		rz_cons_printf("%-12s %-38s %-10s\n", name,
```
Maybe move license before description

PremadeS commented:
> Please add missing authors in corresponding plugins based on the SPDX headers and git history

here? for each file?
`librz/bin/p/bin_any.c`
```bash
RzBinPlugin rz_bin_plugin_any = {
        .name = "any",
        .desc = "Dummy format rz_bin plugin",
        .license = "LGPL3",
        .load_buffer = &load_buffer,
        .destroy = &destroy,
        .baddr = &baddr,
        .info = info,
};
```
*just making sure*

PremadeS committed:
Refactor: move `license` col + change table output

moved license column before description
table output for bin/xtr now contains `author` + `license` cols

PremadeS committed:
Add: author field for `librz/bin/p/*`

author selected on oldest SPDX owner
if same time-period then author is chosen based on git history

PremadeS committed:
Update test for `Print the bin plugins`

PremadeS committed:
Fix: duplicate author + format

PremadeS committed:
Fix: wrong author in test

PremadeS commented:
* added all of the authors, preference given to oldest owner based on `SPDX` header. if same time-frame then decided on git history. Except for the authors of `bin_pyc.c` they both have the same time-frame and none of them are present in git history  *(so the first one in `SPDX` header is chosen)*
* changed the output of bin plugins **table** by adding `author`, `license` and removing `bin/xtr` col, not mentioned in issue but it looked more consistent. can revert back if preferred
* changed the `cmd_list` test to compare the correct output
* *Why is description col in **table** not the last col similar to `rizin -qc 'Lot'` or `rizin -qc 'Lat'`?* because `xtr` plugins don't have author and version fields so it doesn't print right

notxvilka reviewed (approved):
Great!

notxvilka commented:
@PremadeS the only missing part now is better and uniform descriptions of these plugins. Something similar to https://github.com/rizinorg/rizin/pull/5012
Do you want to give it a shot?

PremadeS commented:
> @PremadeS the only missing part now is better and uniform descriptions of these plugins. Something similar to https://github.com/rizinorg/rizin/pull/5012
> Do you want to give it a shot?

Sure
